tools/test/CoverageOutputGenerator/java/com/google/devtools/coverageoutputgenerator/Main.java

```diff
@@ -78,8 +79,13 @@ static int runWithArgs(String... args) throws ExecutionException, InterruptedExc
                 getTracefiles(flags, filesInCoverageDir),
                 LcovParser::parse,
                 flags.parseParallelism()),
-            parseFiles(
-                getGcovInfoFiles(filesInCoverageDir), GcovParser::parse, flags.parseParallelism()));
+	    Coverage.merge(
```

Comment:
I'd only use a single call to `Coverage.merge` here and pass all parsed files to that call. If `merge` only takes two arguments, then change that to take more than two. I think it's strictly better to pass everything in at once, so `Coverage.merge` has the full list - there may be optimizations it can do in that case.